besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/GenerateBlockchainConfig.java

```diff
@@ -242,6 +243,11 @@ private void processExtraData() {
       final String extraData =
           IbftExtraDataCodec.encodeFromAddresses(addressesForGenesisExtraData).toString();
       genesisConfig.put("extraData", extraData);
+    } else if (genesisConfigOptions.isQbft()) {
```

Comment:
on line 239, should a check for null be introduced for `configNode` before continuing?

Reply:
Good catch. I'll add a check for that.